case_studies/starnet/data/sdss/sample_rcfs.txt

```diff
@@ -0,0 +1,309 @@
+RUN=4153 CAMCOL=4 FIELD=100
```

Comment:
Please use the version of the Makefile in the data/sdss directory. It's not necessary to include all these RCFs after all. See `https://github.com/prob-ml/bliss/blob/master/data/sdss/fetch_sample_rcfs.sh`